uint32 seq
int32 group_id
int32[] person_ids
int32[] pi_ids
